app/services/cfe_param_builders/regular_transactions.rb

```diff
@@ -36,7 +36,7 @@ def self.call(income_form, outgoings_form, benefit_details_form = nil, housing_f
     }.freeze
 
     def self.build_payments(cfe_translations, form, operation)
-      cfe_translations.select { |_cfe_name, local_name| form.send("#{local_name}_value").positive? }
+      cfe_translations.select { |_cfe_name, local_name| form.send("#{local_name}_value").to_i.positive? }
```

Comment:
It's that latter one. Sometimes `form.childcare_payments_value` will now be nil, while previously it would always be a number. We want to ignore it if it's nil, so by saying `.to_i.positive?` we change from ruling out `0` to ruling out `nil` AND `0`.